app/routes/organizations/organization/local-involvements/edit.js

```diff
@@ -25,7 +25,7 @@ export default class OrganizationsOrganizationLocalInvolvementsEditRoute extends
       {
         reload: true,
         include:
-          'recognized-worship-type,involvements.involvement-type,involvements.administrative-unit.classification',
+          'recognized-worship-type,involvements.involvement-type,involvements.administrative-unit.classification,involvements.administrative-unit.scope',
```

Comment:
I don't think this is the correct include config.

The save method validates all the involvements and the [organization](https://github.com/lblod/frontend-organization-portal/blob/10ef9b020b8a0cba9de1bec4a095116d2960d2f3/app/controllers/organizations/organization/local-involvements/edit.js#L138) to which we're adding them. It seems the organization has a ["scope" validation](https://github.com/lblod/frontend-organization-portal/blob/10ef9b020b8a0cba9de1bec4a095116d2960d2f3/app/models/administrative-unit.js#L76) which does indeed fail at the moment, but the new code won't fix it as far as I tested.

I think we need to include the scope relationship of the current organization instead?

```suggestion
          'recognized-worship-type,involvements.involvement-type,involvements.administrative-unit.classification,scope',
```

If it worked for you locally I _think_ that might have been by accident, if the scope of the nested administrative units was the same as the organization one by chance?